src/UI/components/map/layers/baseMapList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Collapse from '@mui/material/Collapse';
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+import MapIcon from '@mui/icons-material/Map';
+import {ListButton} from './listButton';
+
+const BaseMapList= ({open, setOpen, openNestBasemapList, setOpenNestBasemapList,
+  sectionTitle, baseMap}:{open:any, setOpen:any, openNestBasemapList:any, setOpenNestBasemapList:any, sectionTitle:string, baseMap:any}) => {

```

Comment:
I don't really like all these `any`s - it's okay to use `any` sometimes, when you're not sure what they type is going to be or if it's too complex to sensibly write out, but it shouldn't be used all the time. Can you update them please?